### DISCUSSION TITLE
Trigger app opening only by keyboard input (not considering mouse movement)

### DISCUSSION THREAD

**Discussion by KeyserSozeZeg** [rxhanson/Charmstone-Community#54] [Ideas]
Would it be possible to add an option, that for app opening the mouse movement is not considered, but only the keyboard inputs are?

To provide some context: I prefer to use keyboard inputs and for simplicity reasons, I use the same modifier key combination for Charmstone as for some actions in Rectangle Pro. I realized that after pressing my configured modifier keys in Charmstone, if my mouse is accidentally moved, then the app opening is triggered, leading to the situation that an incorrect/undesired app is getting opened.

**Comment by rxhanson** [comment]
The initial concept of Charmstone is centered around the cursor movement, but I see what you mean. I'll add this to the backlog. Thanks!

**Comment by szarvas** [comment]
I think my feature request is the same as described above, but I the title is worded strangely. In my wording: **please consider adding an option to disable listening to the mouse**.

I'm using the hyper key + WASD (mapped to arrows by a layer switch) to switch applications. I'm loving it, but as you'd imagine I'm using the hyper key in a lot of other applications.

So I have to take extra care not to accidentally move the mouse when using my hyper shortcuts, otherwise Charmstone gets undesirably activated.

**Comment by rxhanson** [comment]
I realize it's been a while and I haven't added this in. On one hand, it's not a big change to add in a checkbox just for this, but there are actually a lot of configurations like this that I'd like to add in and my plan has been to hold off until I build out the next version of Charmstone that has a lot more going on. 

In the meantime, you can effectively remove the mouse triggering by adjusting parameters via Terminal:

For the non App Store version:
```
defaults write com.knollsoft.CharmstonePro reticleDisplayMinDist -int 10000
defaults write com.knollsoft.CharmstonePro safeAreaSize -int 10000
```

For the App Store version: 
```
defaults write com.knollsoft.Charmstone reticleDisplayMinDist -int 10000
defaults write com.knollsoft.Charmstone safeAreaSize -int 10000
```

Restart Charmstone after.
